tiledb/consolidation_plan.py

```diff
@@ -0,0 +1,59 @@
+import tiledb.cc as lt
+
+from .ctx import Ctx, CtxMixin, default_ctx
+from .libtiledb import Array
+
+
+class ConsolidationPlan(CtxMixin, lt.ConsolidationPlan):
+    """Represents TileDB ConsolidationPlan class"""
```

Comment:
Can we add a more detailed description of what the `ConslidationPlan` actually is?